core/src/test/java/com/opensymphony/xwork2/ognl/OgnlValueStackTest.java

```diff
@@ -238,6 +245,66 @@ public void testFailOnMissingProperty() {
         }
     }
 
+    public void testLogMissingProperties() {
```

Comment:
Since `testNotThrowExceptionOnTopMissingProperty()` added `@since 2.5.21`, do you want the same annotation here ?